intel-retail/automated-self-checkout
# feat: Download video directly from source

## PR Checklist



## What are you changing?

Updating Docker image and downloading video directly from the source.

## Issue this PR will close

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- make run-minikube-demo
- See results being published in automated self checkout inference /tmp/results

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
